pkgs/by-name/sc/scmutils/package.nix

```diff
@@ -0,0 +1,68 @@
+{
+  stdenv,
+  fetchurl,
+  lib,
+  mitschemeX11,
+}:
+stdenv.mkDerivation rec {
```

Comment:
Please use `finalAttrs:` instead of `rec`.

Reply:
OK. I'll push an updated commit.